lib/serializer.js

```
@@ -144,7 +144,14 @@ function parseRelations (data, relations, options) {
     var fk = data[fkName];
 
     var fromType = utils.pluralForModel(relation.modelFrom);
-    var toType = utils.pluralForModel(relation.modelTo);
+    var toType = '';
+    if (relation.polymorphic) {
+      var discriminator = relation.polymorphic.discriminator;
+      var model = options.app.models[data[discriminator]];
+      toType = utils.pluralForModel(model);
+    } else {
+      toType = utils.pluralForModel(relation.modelTo);
+    }
```

The default adapter doesn't hide these properties. Leaving it up to user to manually hide in model definition.